modules/oe_whitelabel_helper/src/Plugin/Block/CorporateEcLogoBlock.php

```diff
@@ -80,15 +81,26 @@ public function build(): array {
     $logo_path = drupal_get_path('module', 'oe_whitelabel_helper') . '/images/logos/ec';
     $title = $this->configFactory->get('system.site')->get('name');
 
-    $build = [
+    $image = [
       '#theme' => 'image',
       '#uri' => $logo_path . '/logo-ec--' . $language . '.svg',
-      '#width' => '240px',
-      '#height' => '60px',
       '#alt' => $title,
       '#title' => $title,
     ];
 
+    $build = [
+      '#type' => 'link',
+      '#url' => Url::fromUri('https://ec.europa.eu/info/index_' . $language, [
```

Comment:
From a maintenance point of view, this path should be placed on a configuration settings. What if the url changes? Then there should be change at git level to put new value here. How is done in OE modules and themes?